signingscript/src/signingscript/sign.py

```diff
@@ -1364,10 +1364,15 @@ async def signer(digest, digest_algo):
     if fmt in ("autograph_authenticode_ev", "gcp_prod_autograph_authenticode_ev", "stage_autograph_authenticode_ev"):
         cafile_key = f"{cafile_key}_ev"
         cert_key = f"{cert_key}_ev"
-    elif fmt.startswith(("autograph_authenticode_202404", "gcp_prod_autograph_authenticode_202404", "stage_autograph_authenticode_202404")):
```

Comment:
This line being replaced actually fixes a possible footgun. With the current code, `stub` formats would not select the `_202404` version of the key. The only reason this hasn't caused problems is because the `authenticode_cert` and `authenticode_ca` values were the same as their `_202404` counterparts.